pywbem/_logging.py

```diff
@@ -32,53 +32,103 @@
 
 * ``pywbem.ops`` logs user-issued calls to pywbem WBEMConnection methods that
   drive WBEM operations (see :ref:`WBEM operations`). Log
-  entries are created at INFO log level. This logs the parameters of each
+  entries are created at DEBUG log level. This logs the parameters of each
   request and the CIMObjects/exceptions in each method repoonse.
 
 * ``pywbem.http`` logs HTTP requests and responses between the pywbem client
-  and WBEM servers, at the info level. This logs the http request data and
+  and WBEM servers, at the DEBUG level. This logs the http request data and
   response data. This named logger is also defined to create log entries when
   logging is at the INFO log level.
 
-To output log records for one of the defined named loggers, either
-:meth:`~pywbem._logging.PywbemLoggers.create_loggers` or the
-:meth:`~pywbem._logging.PywbemLoggers.create_logger` should be used to define
-the characteristics of the named logger.
+The user can use the Python logger functionality to initiate either or both of
+these loggers or helper functions in the _logger module
 
-* :meth:`~pywbem._logging.PywbemLoggers.create_loggers` creates one or more
-  loggers from
-  a string input that defines the component name and characteristics of each
-  logger.  This allows other tools like CLIs that use pywbem to create the
-  pywbem known logs with minimal work from command line or config file input.
+Logging is activated in cim_operations.WBEMConnection with the `enable_logging`
+parameter.
 
-* :meth:`~pywbem._logging.PywbemLoggers.create_logger` creates one or more
-  loggers from
-  parameter inputs that define the component name and characteristics of each
-  logger.
+The following example sets up the operations logger using Python logger
+methods to log to stdout::
 
-These functions save the logger definitions in the
-:class:`~pywbem._logging.PywbemLoggers` class that
-is used by the logging functions in the recorder so trying to create the named
-loggers independently of this code may cause issues.
+    logger = logging.getLogger('pywbem.ops')
+    handler = logging.StreamHandler()
+    handler.setLevel(logging.DEBUG)
 
-The pywbem loggers are based on two parameters (log destination, log detail)
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(message)s')
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+
+    conn.WBEMConnection(... enable_log='all')
+
+When the debug level is set, the pywbem logger will generate log a log entry
+for each WBEMConnection request and response.
```

Comment:
Detail whether this is API call & return vs. CIM-XML request & response.

Reply:
will change all of that with next set of changes